src/SFA.DAS.Apim.Endpoints.sln

```diff
@@ -154,13 +154,15 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Assessors", "Assessors", "{
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Tests", "Tests", "{40E66758-AA64-4939-9DF8-FEC692A27671}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "SFA.DAS.Assessors.Api.UnitTests", "SFA.DAS.Assessors.Api.UnitTests\SFA.DAS.Assessors.Api.UnitTests.csproj", "{8843B30A-F1ED-4FBA-9C65-793C25FB8AEB}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "SFA.DAS.Assessors.Api.UnitTests", "SFA.DAS.Assessors.Api.UnitTests\SFA.DAS.Assessors.Api.UnitTests.csproj", "{8843B30A-F1ED-4FBA-9C65-793C25FB8AEB}"
```

Comment:
I don't think Assessors has changed has it? Maybe revert this if not?

Reply:
this was really odd, it seemed to get changed when i added my project. I reverted a merge from master and accepted their soln, but everytime I added my new class it did this to the assessors project. Really not sure why